packages/gatsby/src/utils/webpack.config.js

```diff
@@ -466,6 +469,81 @@ module.exports = async (
   if (stage === `build-javascript`) {
     const componentsCount = store.getState().components.size
 
+    const splitChunks = {
+      chunks: `all`,
+      cacheGroups: {
+        default: false,
+        vendors: false,
+        framework: {
+          chunks: `all`,
+          name: `framework`,
+          // This regex ignores nested copies of framework libraries so they're bundled with their issuer.
+          test: new RegExp(
+            `(?<!node_modules.*)[\\\\/]node_modules[\\\\/](${FRAMEWORK_BUNDLES.join(
+              `|`
+            )})[\\\\/]`
```

Comment:
mother of regexes! ;)

Possible to do something like that without negative lookbehind with `.*` inside? This trips some tools up (tho it seems to work fine in JS)

Reply:
windows & posix support :'(

Reply:
Yeah, let's not focus on this - not prettiest, but will work